lib/src/timing_specifiers.dart

```diff
@@ -0,0 +1,16 @@
+class StartupTimingType {
```

Comment:
This comment got collapsed without being addressed:

> This class and its constants need doc comments.

Reply:
Added